websites/S/sekolahmu/presence.ts

```diff
@@ -0,0 +1,87 @@
+import { Assets } from 'premid'
+
+const presence = new Presence({
+  clientId: '1037219986378338305',
+})
+const browsingTimestamp = Math.floor(Date.now() / 1000)
+/// ok
+enum ActivityAssets { // Other default assets can be found at index.d.ts
+  Logo = 'https://raw.githubusercontent.com/vintheweirdass/sekolahmu-premid-assets/refs/heads/main/images/logo-v2.png',
+}
+presence.on('UpdateData', () => {
+  // sekolahmu is SSR'ed, so sometimes the previous data can showed up unexpectedly.
+  // we need to make a pause for some milliseconds to prevent that
+  setTimeout(async () => {
+    const presenceData: PresenceData = {
+      largeImageKey: ActivityAssets.Logo,
+      startTimestamp: browsingTimestamp,
+      state: 'Viewing LMS',
+      smallImageKey: Assets.Search,
+    }
+    if (window.document.location.hostname === 'www.sekolah.mu') {
+      if (!window.document.location.pathname.startsWith('/aktivitas')) {
+        if (window.document.location.pathname === '/') {
+          presenceData.state = 'Viewing the homepage'
+        }
+        else if (window.document.location.pathname.startsWith('/rapor')) {
```

Comment:
You can just use `document.location.[...]` 
Considering how many times the pathname is used, also consider destructuring the properties.
```ts
const { pathname, href, search } = document.location
```